### PR TITLE
bug 882719 - Technical/Editorial Review Instructions

### PR DESCRIPTION
**Pull request by emullaney**
For https://bugzilla.mozilla.org/show_bug.cgi?id=882719
Add directions with links specific to editorial/technical review instructions to the review list page.

### CODE REVIEW

File: kuma/wiki/jinja2/wiki/list/needs_review.html [hunk 1]
```diff
@@ -20,6 +20,15 @@
 
       <div id="document-list" class="boxed">
         <h1>{{ title }}</h1>
+        {% if tag|string == 'editorial' %}
+          <p>
+            Please review these articles for spelling, grammar, formatting, layout, etc., and then clear the Editorial review flag. <a href="/en-US/docs/MDN/Contribute/Howto/Do_an_editorial_review#core-steps">See more detailed instructions.</a>
```
Comment (safwanrahman) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: Should be available for localization!

File: kuma/wiki/jinja2/wiki/list/needs_review.html [hunk 2]
```diff
@@ -20,6 +20,15 @@
...
+          </p>
+        {% elif tag|string == 'technical' %}
+          <p>
+            Please review these articles for technical accuracy and completeness, and then clear the Technical review flag. <a href="/en-US/docs/MDN/Contribute/Howto/Do_a_technical_review#core-steps">See more detailed instructions.</a>
```
Comment (safwanrahman) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: Same

File: kuma/wiki/jinja2/wiki/list/needs_review.html (hunk 1 above)
Comment (emullaney) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: Oh right!!! Of course.

File: kuma/wiki/jinja2/wiki/list/needs_review.html [hunk 3]
```diff
@@ -20,6 +20,15 @@
 
       <div id="document-list" class="boxed">
         <h1>{{ title }}</h1>
+        {% if tag|string == 'editorial' %}
+          <p>
+            {{ _('Please review these articles for spelling, grammar, formatting, layout, etc., and then clear the Editorial review flag. <a href="/en-US/docs/MDN/Contribute/Howto/Do_an_editorial_review#core-steps">See more detailed instructions.</a>') }}
+          </p>
```
Comment (jwhitlock) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: Please use this pattern, so the URL will be locale-specific automatically:

```
{% trans url=wiki_url('MDN/Contribute/Howto/Do_an_editorial_review') + '#core-steps' %}
Please review these articles for spelling, grammar, formatting, layout, etc., and then clear the Editorial review flag. <a href="{{ url }}">See more detailed instructions</a>.
{% endtrans %}

File: kuma/wiki/jinja2/wiki/list/needs_review.html [hunk 4]
```diff
@@ -20,6 +20,15 @@
...
+        {% elif tag|string == 'technical' %}
+          <p>
+            {{ _('Please review these articles for technical accuracy and completeness, and then clear the Technical review flag. <a href="/en-US/docs/MDN/Contribute/Howto/Do_a_technical_review#core-steps">See more detailed instructions.</a>') }}
```
Comment (jwhitlock) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: nitpick - place the period outside the </a> tag

**Comment by jwhitlock** [comment]
Practical tests work in my VM, looks good. A couple more changes to make:

* [ ] Use ``{% trans %}`` tag and the ``wiki_url`` helper
* [ ] Move that period

I've updated [bug 882719](https://bugzilla.mozilla.org/show_bug.cgi?id=882719), in case someone wants to argue about periods and ``<a>`` tags.

**Comment by emullaney** [comment]
Made changes requested. Thanks for all the pointers, @jwhitlock!

- [x] Use {% trans %} tag and the wiki_url helper
- [x] Move that period

File: kuma/wiki/jinja2/wiki/list/needs_review.html [hunk 5]
```diff
@@ -20,6 +20,20 @@
 
       <div id="document-list" class="boxed">
         <h1>{{ title }}</h1>
+        {% if tag|string == 'editorial' %}
+
```
Comment (jwhitlock) [inline kuma/wiki/jinja2/wiki/list/needs_review.html]: nitpick - can you clean up this extra whitespace?

**Comment by emullaney** [comment]
- [x] Extraneous carriage return removed.

**Pull request merged**